src/rgb/standards.rs

```diff
@@ -0,0 +1,53 @@
+//!Various RGB standards.
+
+use num::Float;
+use Yxy;
+use flt;
+use white_point::{WhitePoint, D65};
+use rgb::{Primaries, RgbSpace, RgbStandard};
+use pixel::TransferFn;
+
+///The sRGB color space.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub struct Srgb;
+
+impl Primaries for Srgb {
```

Comment:
If I am not mistaken easyrgb and lindbloom calculators adapt the primaries, so I think  chromatic adaptation is the way to go.

One issue I currently see is the the AdaptFrom does not check if the source and destination white points are equal and will cause D65 -> D65 changes. I guess we should add a test for that.